Modules/Core/Common/include/itkPointSetToImageFilter.hxx

```diff
@@ -130,7 +130,7 @@ PointSetToImageFilter<TInputPointSet, TOutputImage>::GenerateData()
   for (i = 0; i < InputPointSetDimension; ++i)
   {
     size[i] = static_cast<SizeValueType>(bb->GetBounds()[2 * i + 1] - bb->GetBounds()[2 * i]);
-    origin[i] = 0; // bb->GetBounds()[2*i];
+    origin[i] = bb->GetBounds()[2 * i];
   }
```

Comment:
Probably setting the output image's properties (origin, spacing, etc.), i.e. a few dozen lines below, should dwell in a separate e.g. protected method for an improved encapsulation. Left for a separate PR.